contracts/Tokens/VTokens/VToken.sol

```diff
@@ -495,7 +509,8 @@ contract VToken is VTokenInterface, Exponential, TokenErrorReporter {
         _notEntered = true;
 
         isFlashLoanEnabled = flashLoanEnabled_;
-        flashLoanFeeMantissa = flashLoanFeeMantissa_;
+        flashLoanProtocolFeeMantissa = flashLoanProtocolFeeMantissa_;
+        flashLoanSupplierFeeMantissa = flashLoanSupplierFeeMantissa_;
```

Comment:
Should we check any constraints for these variables? Maximum fee?

Reply:
What should be the values for the max threshold?

Reply:
Yes, we can, I guess, for `flashLoanProtocolFeeMantissa` max could be `0.1%` and for we should be open for ` flashLoanSupplierFeeMantissa`. We can discuss about the values, let me know your thoughts.